nodewatcher/nodewatcher.py

```diff
@@ -142,6 +132,22 @@ def maintainSize(region, asg):
         log.debug('capacity less then or equal to min size.')
         return True
 
+
+def saveIdleTime(persisted_data):
+    try:
+        if not os.path.exists('/var/run/nodewatcher/'):
```

Comment:
I'd suggest to define a couple of module _"constants"_ , something like
_NW_DATA_DIR = "/var/run/nodewatcher/"
_NW_IDLETIME_FILE  = _NW_DATA_DIR + "node_idletime.json"
and then refer them in the code.

Reply:
Okay